test-end-to-end/test-descriptions/002_001_admin_add_new_course_and_teacher_test_description.md

```diff
@@ -0,0 +1,43 @@
+# Admin add new course and new members to that course
```

Comment:
This scenario should not use bulk import - this is the scenario that adding users to a course manually using the UI.